app/gateway/database/postgres/accounts/create_account.go

```diff
@@ -23,7 +22,7 @@ func (repository accountRepository) Create(ctx context.Context, account *account
 		account.Created_at)
 
 	if err != nil {
-		return nil, errorCreateAccount
+		return account, err
```

Comment:
uma prática que costumamos fazer é retornar um `account.Account{}` (uma estrutura vazia) ao invés de um objeto com valores, para não confudir quem chamar o método e for retornado dois valores válidos